buildman/manager/executor.py

```diff
@@ -671,6 +671,58 @@ def stop_builder(self, builder_id):
             raise ExecutorException("Failed to send delete pod call for job %s", builder_id)
 
 
+class KubernetesPodmanExecutor(KubernetesExecutor):
+    def __init__(self, *args, **kwargs):
+        super(KubernetesExecutor, self).__init__(*args, **kwargs)
+        self.namespace = self.executor_config.get("BUILDER_NAMESPACE", "builder")
+        self.image = self.executor_config.get(
+            "BUILDER_CONTAINER_IMAGE", "quay.io/quay/quay-builder-podman:stable"
+        )
+
+    def _build_job_containers(self, token, build_uuid):
+        server_grpc_addr = (
+            self.manager_hostname.split(":", 1)[0] + ":" + str(SECURE_GRPC_SERVER_PORT)
+            if self.registry_hostname == self.manager_hostname
+            else self.manager_hostname
+        )
+
+        container = {
+            "name": "builder",
+            "imagePullPolicy": "Always",
+            "image": self.image,
+            "env": [
+                {"name": "TOKEN", "value": token},
+                {"name": "BUILD_UUID", "value": build_uuid},
+                {"name": "SERVER", "value": server_grpc_addr},  # manager_hostname
+                {"name": "REGISTRY_HOSTNAME", "value": self.registry_hostname},
+                {"name": "CONTAINER_RUNTIME", "value": "podman"},
+                {"name": "BULDAH_ISOLATION", "value": "chroot"},
+                {"name": "CA_CERT", "value": self.executor_config.get("CA_CERT", self._ca_cert())},
+                {"name": "TLS_CERT_PATH", "value": "/certs/cacert.crt"},  # Used by build manager
+                {
+                    "name": "SSL_CERT_FILE",
+                    "value": "/certs/cacert.crt",
+                },  # Used for other Podman and other Go libraries
```

Comment:
One is picked up by Go and the other by the quay-builder. I could modify the quay-builder to pick up on the same env var as Go, it would just have to listen for both to make sure it's backwards compatible

Reply:
I think it's fine. Should just add a note on which one is used when both are set.